docs/book/src/quinn/data-transfer.md

```diff
@@ -0,0 +1,172 @@
+# Data Transfer
+
+In the [previous chapter](set-up-connection.md) we characterized how to set up an [Endpoint][Endpoint]
+and then get access to a [NewConnection][NewConnection] instance.
+Now we will continue with the subject of sending data over this connection.
+
+## Multiplexing
+
+A QUIC stream can be compared to a TCP stream, eventhough if it is based on UDP. 
+The big difference is that you are not limited to a single stream. 
+You can open multiple streams between two peers, this is also called multiplexing.
+
+Stream multiplexing can have a significant positive effect on application performance if the resources allocated to streams are properly prioritized.
+Currently multiplexing it is used in a protocol like HTTP/2, but unlike HTTP multiplexing, QUIC does not automatically multiplex.
+This is an advantage because the user gets full control over the multiplexing. 
+Another difference compared to a TCP stream is that QUIC is not limited to one kind of stream but can open both bidirectional and unidirectional streams.
```

Comment:
I think TCP sockets can close one side of the connection, which also results in a unidirectional stream? So this is not quite accurate. I'd just leave the TCP comparison out here and briefly explain that there are four kinds of streams for each QUIC connection: client/server-initiated bi/uni-directional.

Reply:
Good one!